arena.go

```diff
@@ -0,0 +1,39 @@
+package bigqueue
+
+import (
+	"os"
+	"syscall"
+
+	"github.com/grandecola/mmap"
+)
+
+const (
+	cFilePerm = 0777
```

Comment:
Exec permission is not required. Also other users should have just read access. We can discuss this.